pkg/systemd/dbus.go

```diff
@@ -129,7 +129,11 @@ func dbusAuthRootlessConnection(createBus func(opts ...godbus.ConnOption) (*godb
 func newRootlessConnection() (*dbus.Conn, error) {
 	return dbus.NewConnection(func() (*godbus.Conn, error) {
 		return dbusAuthRootlessConnection(func(opts ...godbus.ConnOption) (*godbus.Conn, error) {
-			path := filepath.Join(os.Getenv("XDG_RUNTIME_DIR"), "systemd/private")
+			path := filepath.Join(os.Getenv("XDG_RUNTIME_DIR"), "systemd", "private")
+			path, err := filepath.EvalSymlinks(path)
```

Comment:
And, like above, I think you need an existence check before you `EvalSymlinks`.

Reply:
What would the check do if it does not exists?

Still call
                        return godbus.Dial(fmt.Sprintf("unix:path=%s", path))
?

 If the file does not exists EvalSymlinks will return that it does not exists.

Reply:
I don't know enough about this `systemd/private` path to comment. But:
* old code: just returns a string with the path, whether the path exists or not (leave it to caller to deal with problems)
* new code: barfs if path does not exist. This is new behavior. I don't know if it's good new behavior or bad new behavior... but it's a change.

Reply:
I think the old code will barf when godbus.Dial attempts to connect to a non existing socket
The new code will report the non existing socket earlier, as I read it.